doc/proxy-metrics.md

```diff
@@ -38,52 +38,69 @@ request headers are received to when the response stream has completed.
 
 Each of these metrics has the following labels:
 
+* `authority`: The value of the `:authority` (HTTP/2) or `Host` (HTTP/1.1)
+               header of the request.
+* `direction`: `inbound` if the request originated from outside of the pod,
+               `outbound` if the request originated from inside of the pod.
+
+### Response Labels
+
+The following labels are only applicable on `response_*` metrics.
+
 * `classification`: `success` if the response was successful, or `failure` if
                     a server error occurred. This classification is based on
                     the gRPC status code if one is present, and on the HTTP
                     status code otherwise. Only applicable to response metrics.
-* `direction`: `inbound` if the request originated from outside of the pod,
-               `outbound` if the request originated from inside of the pod.
-* `authority`: The value of the `:authority` (HTTP/2) or `Host` (HTTP/1.1)
-               header of the request.
-* `dst_deployment`: The deployment to which this request is being sent.  Only
-                    applicable if `direction=outbound`.
-* `dst_job`: The job to which this request is being sent.  Only applicable if
-             `direction=outbound`.
-* `dst_replica_set`: The replica set to which this request is being sent.  Only
-                     applicable if `direction=outbound`.
-* `dst_daemon_set`: The daemon set to which this request is being sent.  Only
-                    applicable if `direction=outbound`.
-* `dst_replication_controller`: The replication controller to which this request
-                                is being sent.  Only applicable if
-                                `direction=outbound`.
-* `dst_namespace`: The namespace to which this request is being sent.  Only
-                   applicable if `direction=outbound`.
-* `status_code`: The HTTP status code of the response.  Only applicable to
-                 response metrics.
 * `grpc_status_code`: The value of the `grpc-status` trailer.  Only applicable
-                      to response metrics for gRPC responses.
+                      for gRPC responses.
+* `status_code`: The HTTP status code of the response.
+
+### Outbound labels
+
+The following labels are only applicable if `direction=outbound`.
```

Comment:
should we also mention that they're only available on `response_*` metrics? since the Outbound Labels are on the same top level as Response Labels that info is lost.

Reply:
nvm! I was confused. ignore this! :)